docs/overview/status/ProjectStatus.stories.mdx

```diff
@@ -5,16 +5,16 @@ import data from "./data.json";
 
 # Project Status
 
-Here you can stay up-to-date on the status of project and its development. This can help users of the UI kit understand what features they can currently take advantage of, as well as what they can expect to see in future updates.
+Stay up-to-date on the status of project and its development. This can help users of the UI kit understand what features they can currently take advantage of, as well as what they can expect to see in future updates.
```

Comment:
```suggestion
Stay up-to-date on the status of the project and its development. This can help users of the UI kit understand which features they can currently take advantage of, as well as what they can expect to see in future updates.
```